src/Jackalope/Transport/Jackrabbit/Client.php

```diff
@@ -271,10 +273,30 @@ public function sendExpect($send = true)
      * Set to true to force HTTP version 1.0
      *
      * @param boolean
+     *
+     * @deprecated use addCurlOptions([CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_0]) instead
      */
     public function forceHttpVersion10($forceHttpVersion10 = true)
     {
-        $this->forceHttpVersion10 = $forceHttpVersion10;
+        if ($forceHttpVersion10) {
+            $this->addCurlOptions(array(CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_0));
+        } elseif (array_key_exists(CURLOPT_HTTP_VERSION, $this->curlOptions)) {
+            unset($this->curlOptions[CURLOPT_HTTP_VERSION]);
```

Comment:
no need for the array_key_exists check, just unset right away. unsetting something that does not exist is no error.